doc/en/assert.rst

```diff
@@ -85,6 +85,13 @@ and if you need to have access to the actual exception info you may use::
 the actual exception raised.  The main attributes of interest are
 ``.type``, ``.value`` and ``.traceback``.
 
+In the context manager form you may use the keyword argument
```

Comment:
Hmm that's a good question. The other places in pytest documentation which use this feature put it before the paragraph...

You are right, that's how Python's own documentation does it at least. :grin: